src/node_http2_state.h

```diff
@@ -21,6 +21,9 @@ namespace http2 {
     IDX_SETTINGS_COUNT
   };
 
+  // number of max additional settings, thus settings not implemente by nghttp2
```

Comment:
```suggestion
  // number of max additional settings, thus settings not implemented by nghttp2
```

Reply:
Ok, changed on my development copy, will commit and push later all together.